### PR TITLE
LA-8905 Looker permission check script

### PR DESCRIPTION
**Pull request by kaif-lb**
- added a script to check user's permission check for looker datasource

### CODE REVIEW

File: sql_user_check_looker/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Check permissions on Looker instance
+
+
+### Run the script
+
+Running the script will print models and explores count which user has access to. Looker instance url, client_id and client_secret
+are inputs to this script. URL looks something like this https://23151ff5-b9c5-4327-9095-2cdaa11c1e43.looker.app
```
Comment (arpit-lb) [inline sql_user_check_looker/README.md:7]: Remove the url and give a template like <instanceId>.looked.app

File: sql_user_check_looker/run.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
```
Comment (arpit-lb) [inline sql_user_check_looker/run.sh:4]: Check if jq and curl are present. Raise error accordingly

File: sql_user_check_looker/run.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+readonly URL=$URL
+readonly CLIENT_ID=$CLIENT_ID
+readonly CLIENT_SECRET=$CLIENT_SECRET
+
+response=$(curl -s -d "client_id=$CLIENT_ID&client_secret=$CLIENT_SECRET" "$URL/api/4.0/login")
```
Comment (arpit-lb) [inline sql_user_check_looker/run.sh:9]: Check if status code was 200. Raise error otherwise. Same comment for other occurences

**Review by arpit-lb — CHANGES_REQUESTED** [review]

File: sql_user_check_looker/run.sh (hunk 2 above)
Comment (kaif-lb) [reply sql_user_check_looker/run.sh:4]: done.

File: sql_user_check_looker/README.md (hunk 1 above)
Comment (kaif-lb) [reply sql_user_check_looker/README.md]: done.

File: sql_user_check_looker/run.sh (hunk 3 above)
Comment (kaif-lb) [reply sql_user_check_looker/run.sh]: done.

**Review by arpit-lb — APPROVED** [review]

**Pull request merged**